PubHub/PubHub.BookMobile/MauiProgram.cs

```diff
@@ -34,8 +39,17 @@ public static MauiApp CreateMauiApp()
                     options.HttpClientName = ApiConstants.HTTPCLIENT_NAME;
                     options.AppId = builder.Configuration.GetSection("ApiSettings").GetValue<string>(ApiConstants.APP_ID) ?? throw new NullReferenceException("Application ID couldn't be found.");
                     options.ConfigureForMobile = true;
+                    options.TokenInfoAsync = (provider) =>
+                    {
+                        if (User.TryGetCachedToken(out TokenInfo? result))
+                            return Task.FromResult(User.GetChachedToken());
+                        return Task.FromResult(new TokenInfo { RefreshToken = string.Empty, Token = string.Empty });
+                    };
                 });
 
+            //  Remove underline from entries on Android
+            Microsoft.Maui.Handlers.EntryHandler.Mapper.AppendToMapping(nameof(Entry), (handler, view) => handler.PlatformView.BackgroundTintList = ColorStateList.ValueOf(Android.Graphics.Color.Transparent));
```

Comment:
Did you fix the missing login boxes on the login page in the app? - and did this had something to do with it? 🤔

Reply:
That could very well be the problem to be honest 🤔

Reply:
Should have fixed this now